### PR TITLE
126 new icons, 1 update

### PR DESCRIPTION
**Pull request by x9136**
## Type of change
:x: Bug fix
:white_check_mark: Icon addition
:x: General change
### Updated
Moon+ Reader Pro (`com.flyersoft.moonreaderp/com.flyersoft.moonreaderp.ActivityMain`)
Moon+ Reader Pro (`com.flyersoft.moonreaderp/com.flyersoft.moonreaderp.ActivityTxt`)
### Requests
Alert (`com.ukrainealarm/com.ukrainealarm.main.MainActivity`)
AndDrop (`com.neure.anddrop/com.neure.anddrop.SplashActivity`)
Anytype Legacy (`com.anytypeio.anytype/com.anytypeio.anytype.ui.main.MainActivity`)
App Locker Master (`com.vnsolutions.fastappslocker/com.gmsolution.fastapplocker.MainActivityAlias`)
Arena Kuota (`com.arenakuota.mobile/com.w38s.StartupActivity`)
Artifact (`ai.nokto.wire/ai.nokto.wire.MainActivity`)
AstraCrypt (`com.nevidimka655.astracrypt/com.nevidimka655.astracrypt.MainActivity`)
Auto Change Wallpaper (`viet.dev.apps.autochangewallpaper/viet.dev.apps.autochangewallpaper.activities.SplashActivity`)
Autosync (`com.ttxapps.autosync/com.ttxapps.sync.app.MainActivity`)
Bar and Bench (`com.barandbench/com.zoontek.rnbootsplash.RNBootSplashActivity`)
Battery Widget (`com.lstapps.batterywidget/com.lstapps.batterywidget.MainActivity`)
BC Services Card (`ca.bc.gov.id.servicescard/ca.bc.gov.id.servicescard.screens.splash.SplashActivity`)
Bibit (`com.bibit.bibitid/com.bibit.bibitid.ui.BibitActivity`)
Bitexen Global (`com.bitexenglobal.exchangeapp/com.bitexenglobal.exchangeapp.activity.SplashScreenActivity`)
BMO Digital Banking (`com.bmoharris.digital/com.bmoharris.digital.MainActivity`)
Booksy (`net.booksy.customer/net.booksy.customer.SplashActivity`)
Botim (`im.thebot.messenger/im.thebot.messenger.BOTStartPage`)
BSRemote (`net.froemling.bsremote/net.froemling.bsremote.ScanActivity`)
Caixadirecta (`cgd.pt.caixadirectaparticulares/cgd.pt.caixadirectaparticulares.MainActivity`)
Caustic (`com.singlecellsoftware.caustic/com.singlecellsoftware.caustic.CausticActivity`)
Chase UK (`com.chase.intl/com.mobile.MainActivity`)
CIC (`com.cic_prod.bad/com.e_i.bad.controls.activities.StartActivity`)
CIMB Bank PH (`com.cimbph.app2022/com.ocft.SplashActivity`)
Click Up (`air.com.ssdsoftwaresolutions.clickuz/uz.click.evo.ui.mainrouter.MainRouterActivity`)
Clip Stack (`com.catchingnow.tinyclipboardmanager/com.catchingnow.tinyclipboardmanager.activity.ActivityMain`)
Copy (`com.weberdo.apps.copy/com.weberdo.apps.copy.MainActivity`)
CrontoSign Swiss (`com.cronto.crontosign.swiss/com.onespan.jemmic.crontosignswiss.feature.splash.SplashNotificationActivity`)
Daijishō (`com.magneticchen.daijishou/com.magneticchen.daijishou.activities.BootstrapActivity`)
DailyObjects (`com.dailyobjects/com.dailyobjects.marche.main.MainActivity`)
DeadHash (`com.codedead.deadhash/com.codedead.deadhash.gui.MainActivity`)
Denník N (`sk.dennikn.dennikn/com.dennikn.android.dennikn.MainActivity`)
Devialet (`io.dvlt.blaze/io.dvlt.blaze.StartActivity`)
DioHub (`com.felix.diohub/com.felix.diohub.MainActivity`)
DroidRec (`com.yakovlevegor.DroidRec/com.yakovlevegor.DroidRec.MainActivity`)
Easycash (`com.fintopia.idnEasycash.google/com.lingyue.easycash.activity.launch.EasyCashLaunchActivity`)
e﻿I﻿D Me (`ca.bluink.eid_me_and/ca.bluink.eid_me_and.MainActivity`)
FaceShow (`com.picastory.ins.editor/com.ufotosoft.home.SplashActivity`)
Filen (`io.filen.app/io.filen.app.MainActivity`)
Fine Lock (`yuh.yuh.finelock/yuh.yuh.finelock.T`)
Frequaw (`com.jujinkim.frequaw/com.jujinkim.frequaw.ui.MainActivity`)
Gaia GPS (`com.trailbehind.android.gaiagps.pro/com.trailbehind.activities.MainActivity`)
GasBuddy (`gbis.gbandroid/gbis.gbandroid.MainActivity`)
GeoGuessr (`com.geoguessr.app/com.geoguessr.app.ui.StartActivity`)
GKB Bank (`com.netcetera.android.mbanking.gkb/com.netcetera.android.mbanking.base.controller.SplashScreenActivity`)
GoArt (`com.everimaging.goart/com.everimaging.goart.SplashActivity`)
GymKeeper (`com.kg.app.sportdiary/com.kg.app.sportdiary.activities.MainActivity`)
Harmonoid (`com.alexmercerind.harmonoid/com.alexmercerind.harmonoid.MainActivity`)
HEY (`com.basecamp.hey/com.basecamp.hey.main.MainActivity`)
Home options (`com.ss.squarehome2/com.ss.squarehome2.MyPreferencesActivity`)
HTTP Custom (`xyz.easypro.httpcustom/team.dev.epro.apkcustom.MainActivity`)
Huebsch (`com.alliancelaundry.app.huebsch/com.alliancelaundry.app.activities.SplashActivity`)
Hydro-Québec (`com.hydroquebec.mf_android/com.hydroquebec.mf_android.MainActivity`)
iBetter (`com.clover.ibetter/com.clover.ibetter.ui.activity.MainActivity`)
imagin (`com.imaginbank.app/com.imaginbank.appima.darkimagin`)
inshorts (`com.nis.app/com.nis.app.ui.activities.HomeActivity`)
IP Tools (`com.ddm.iptools/com.ddm.iptools.ui.MainActivity`)
Itaú Cartões (`com.itaucard.activity/br.com.itau.cartoes.presentation.splash.SplashActivity`)
JustAuthMe (`me.justauth.app.android/host.exp.exponent.MainActivity`)
Kanton Internet (`com.kantoninternet.customer/com.kantoninternet.customer.MainActivity`)
Kiwibank (`nz.co.kiwibank.mobile/nz.co.kiwibank.mobile.ui.others.activities.SplashActivity`)
League (`life.league/life.league.base.LaunchActivity`)
LINE MAN (`com.linecorp.linemanth/net.appsynth.lineman.view.splash.SplashActivity`)
MegaSync (`com.ttxapps.megasync/com.ttxapps.sync.app.MainActivity`)
Message Plus (`com.mezo/com.mezo.TestTabs.ActivityBlockVer99`)
Migros (`ch.migros.app/ch.migros.app.misc.MasterActivity`)
Mivo (`ins.story.unfold/com.ufotosoft.home.SplashActivity`)
Mobiuz (`uz.mobiuz.mobiservice/uz.mobiuz.mobiservice.dev.SplashActivity`)
Money Dashboard (`com.mdb.mdbandroidneon/com.mdb.mdbandroidneon.SplashActivity`)
MorseMania (`net.countrymania.morse/net.countrymania.morse.MainActivity`)
Mumbai 1 (`com.mmrda.mumbaione/com.trufaremobi.mmrda.view.activities.SplashActivity`)
MVV-App (`com.mdv.companion/com.mdv.companion.MainActivity`)
Nawgati (`in.cng.ecoconnect/in.cng.ecoconnect.MainActivity`)
Nearby Share (`com.cooliehat.nearbyshare/com.cooliehat.nearbyshare.sharingmodule.Activities.SplashActivity`)
neon bank (`com.neonbanking.app/com.neonbanking.app.MainActivity`)
Notification History Log (`com.ikvaesolutions.notificationhistorylog/com.ikvaesolutions.notificationhistorylog.views.activity.SplashScreenActivity`)
Notification Widget (`com.kuma.notificationwidget/com.kuma.notificationwidget.MainActivity`)
Official TWRP App (`me.twrp.twrpapp/me.twrp.officialtwrpapp.activities.MainActivity`)
OKX (`com.okinc.okex.gp/com.okinc.okex.welcome.WelcomeActivity`)
One4Wall (`one4studio.wallpaper.one4wall/one4studio.wallpaper.one4wall.MainActivity`)
Openbank (`es.openbank.mobile/com.accountmanager.myapplication.views.activities.LoginActivity`)
Papers (`com.wallpapers.papers/com.wallpapers.papers.activity.SplashScreen`)
PDF Converter (`swati4star.createpdf/swati4star.createpdf.activity.SplashActivity`)
Permission Ruler (`com.stefanosiano.permissionruler/com.stefanosiano.permissionruler.view.activity.M10MainActivity`)
PhotoShot (`com.vyroai.photoeditorone/com.vyroai.photoeditorone.ui.MainActivity`)
Piktures (`com.diune.pictures/com.diune.pikture_ui.ui.main.MainActivity`)
Playtomic (`com.playtomic/com.playtomic.general.LaunchScreenActivity`)
Plum (`com.withplum.app/com.plum.app.features.intro.SplashActivity`)
Podcast Addict (`com.bambuna.podcastaddict/com.bambuna.podcastaddict.activity.PodcastListActivity`)
Premise (`com.premise.android.prod/com.premise.android.activity.launch.LaunchActivity`)
Psono (`com.psono.psono/com.psono.psono.MainActivity`)
Punto (`com.punto.punto/com.punto.punto.MainActivity`)
Quinyx (`quinyx.mobile/quinyx.mobile.kotlin.presentation.splash.StartActivity`)
Quotex (`io.quotex/io.quotex.webview.MainActivity`)
RadioPlay (`fi.bauermedia.rpf/fi.bauermedia.rpf.BauerIntroActivity`)
Railcard (`com.raildeliverygroup.railcard/com.raildeliverygroup.railcard.Launcher`)
Roll20 (`net.roll20.player/net.roll20.player.MainActivity`)
RxLocal (`com.newtechsys.rxlocalmobile/crc64cf08890f329bc7ff.MainActivity`)
Screen Recorder by naveensingh (`com.linuxauthority.screenrecorder/com.linuxauthority.screenrecorder.ui.main.MainActivity`)
SHARE NOW (`com.car2go/activity.MainActivity`)
Smart-ID (`com.smart_id/com.stagnationlab.sk.MainActivity`)
SMS Backup and Restore (`com.riteshsahu.SMSBackupRestore/com.riteshsahu.SMSBackupRestore.activities.IntroActivity`)
Sofascore (`com.sofascore.results/com.sofascore.results.main.StartActivity`)
Square Home (`com.ss.squarehome2/com.ss.squarehome2.MainActivity`)
SuperBeam (`com.majedev.superbeam/com.majedev.superbeam.activities.main.MainIntermediateActivity`)
SVPlayer (`com.svpteam.svp/com.svpteam.SVPActivity`)
Sync for Lemmy (`io.syncapps.lemmy_sync/io.syncapps.lemmy_sync.ui.activities.HomeActivity.Default`)
TEO (`com.comeco.teo/crc64753b843c3502ab2c.SplashActivity`)
Testy (`com.robertlevonyan.testy/com.robertlevonyan.testy.base.MainActivity`)
THE COUPLE (`com.ibillstudio.thedaycouple/com.ibillstudio.thedaycouple.activity.SplashActivity`)
The Race (`com.theracemadialtd.therace/crc643f8e59ff9f999875.SplashActivity`)
TICE (`app.tice.TICE.production/tice.ui.activitys.MainActivity`)
TransitDB (`ca.transitdb.mobile.android/ca.transitdb.mobile.android.MainActivity`)
Unitto (`com.sadellie.unitto/com.sadellie.unitto.MainActivity`)
Vaulty (`com.theronrogers.vaultyfree/com.vaultyapp.login.LoginActivity`)
Viettel Money (`com.bplus.vtpay/com.bplus.vtpay.activity.SplashScreen`)
VRidge (`com.riftcat.vridge2/com.riftcat.vridge.MainActivity`)
VRidge (Diagnostics) (`com.riftcat.vridge2/com.riftcat.vridge.DiagnosticEntry`)
Wake on Lan (`de.florianisme.wakeonlan/de.florianisme.wakeonlan.ui.MainActivity`)
WhatsAuto (`com.guibais.whatsauto/com.guibais.whatsauto.HomeActivity`)
Wish Explorer (`com.wishexplorer/com.wishexplorer.MainActivity`)
Woocer (`com.woocer.woocommerceapp/com.woocer.woocommerceapp.ui.intro.SplashActivity`)
x-kom (`pl.xkom/pl.xkom.ui.MainActivity`)
Yandex Metro (`ru.yandex.metro/ru.yandex.metro.LaunchActivity`)
YASAN Launcher (`yasan.space.mnml.ai.launcher/dev.yasan.launcher.legacy.ui.main.MainActivity`)
Русский язык. Орфография (`com.greenkeycompany.exam/com.greenkeycompany.exam.main.activity.view.MainActivity`)
Фора (`ua.fora.android/ua.silpo.android.ui.activity.DynamicLinksRouteActivity`)
出光公式アプリ  (`jp.showa_shell_dmp.shellpass/jp.co.dropsystem.shell_android.MainActivity`)

### CODE REVIEW

File: svgs/chase_uk.svg
Comment (x9136) [inline svgs/chase_uk.svg:1]: This is intentional to keep the ability to distinguish between apps.

File: svgs/xkom.svg
Comment (flame-0) [inline svgs/xkom.svg:1]: I think you forgot to remove the opacity here

Comment (x9136) [reply svgs/xkom.svg:1]: Nah, it's okay. After downloading the svg do you get the background somehow?

File: svgs/chase_uk.svg
Comment (KTibow) [reply svgs/chase_uk.svg:1]: Why would someone have both apps on their phone?

Comment (x9136) [reply svgs/chase_uk.svg:1]: @KTibow If it is possible to add a difference, but keep the essence, it is better not to be lazy and do it, than to allow the possibility of identical icons. For example, recently 3 completely different apps from Adobe had the same icon. It was impossible to figure out which one was which, if you prefer to hide the names of the apps for a cleaner screen.

As for the icon. Only people who travel between US and UK and have JPMorgan accounts will have two apps. It's unlikely that any such people use Lawnicons, although there have been requests for both apps separately. So I care about the theoretical situation. Why not, to save 500 bytes?

**Comment by x9136** [comment]
@SuperDragonXD I'm not quite sure what new problems I'm going to have to deal with now. Can you explain?

**Comment by KTibow** [comment]
Just properly format the appfilter xml so that the whitespace is consistent

**Comment by x9136** [comment]
> Just properly format the appfilter xml so that the whitespace is consistent

How did it happen that such a task has to be done manually?

**Comment by KTibow** [comment]
Because people previously added icons with improper indentation
Also it shouldn't have to be manual - just find an XML formatter online, set the indent size to 2 spaces, and format what you have right now. Or just reset your appfilter to the upstream one and readd the icons, as there's now no improper indentation

**Comment by x9136** [comment]
Feel free to download all the icons and make your own PR. 😉

**Pull request closed without merging**